http/http_test.go

```diff
@@ -16,6 +16,9 @@ import (
 )
 
 func testHttpGet(t *testing.T, token string, addr string) *http.Response {
+	if len(token) == 0 {
+		token = "<empty>"
```

Comment:
This change fundamentally alters the way `testHttpData` works. This likely shouldn't be necessary. I'm guessing what you want to do is actually modify testHttpData to always include the header even if there is no actual value.

Reply:
Yeah my bad, I made this change purely for `t.Logf` right below. Somehow I completely missed that this value gets passed into `testHttpData` even though it's clearly in there...